### ISSUE TITLE
[Bug] Hit/Miss always hidden

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#628]
In GitLab by @carl.steib on Dec 28, 2021, 18:04

When using Foundry v9 (build 238) and midi-qol 0.8.98, the hit/miss card always shows as private to the GM, even when "Check - all see result" is selected for the hit auto check.

On Foundry 0.8.9 and midi-qol 0.8.90, using the same configuration settings, this issue does not occur.

**Comment by tposney** [comment]
Can you export your midi-qol settings? I can't repro the problem so need to see you settings.

**Comment by tposney** [comment]
I think I've found the problem. In 0.9, for reasons that escape me, the public roll mode can now be (apparently randomly) be set to "roll" or "publicroll". I did not realise such a change had been made and only check for "roll".

If you do a private roll and then revert back to public the rollmode gets set to "publicroll", but otherwise is "roll". Which explains why I did not discover it.

Anyway fix will go out in 0.8.101
